### ISSUE TITLE
ENHANCEMENT: Parsing fields manually should have generic helpers

### ISSUE THREAD

**Issue by marchy** [ngocnicholas/airtable.net#65]
After the move to System.Text.Json, parsing out fields manually has become really cumbersome due to an extra cast to `JsonElement` needed, which has to be wrapped in braces (due to the forced casting syntax) and then the new `.GetString()` / `GetBoolean()` etc. methods used to parse the appropriate values.

This also leaks the System.Text.Json abstraction outwards through the Airtable.NET library, which is further out than it ought to be.

**Example:**
```
string? name = ((JsonElement?)GetField( "Name" ))?.GetString()
```

Instead I propose generic `GetField<T>` methods added that hide away all the casting complexity – and indeed hide away the System.Text.Json abstraction altogether.

**Suggested:**
```
string? name = regionRecord.GetField<string?>( "Name" )
```

**Comment by marchy** [comment]
I've submitted up a PR that implements helpers for both primitive fields as well as arrays, lists, collections and enumerables, while also exposing an `GetFieldAsJson(..)` if you do want a convenience to access the System.Text.Json `JsonElement` directly (as opposed to getting an `object` back that you have to cast each time.

**Comment by marchy** [comment]
Has this been pushed out in a new version yet by chance?

**Comment by ngocnicholas** [comment]
A new version of Airtable.net will be released next week.

**Comment by ngocnicholas** [comment]
Enhancements included in new release of 1.3.0.
